datalad/core/distributed/tests/test_clone.py

```diff
@@ -505,8 +506,13 @@ def test_cfg_originorigin(path):
     # Clone another level, this time with a relative path. Drop content from
     # lev2 so that origin is the only place that the file is available from.
     clone_lev2.drop("file1.txt")
-    with chpwd(path):
+    with chpwd(path), swallow_logs(new_level=9) as cml:
         clone_lev3 = clone('clone_lev2', 'clone_lev3')
+        # we called git-annex-init; see gh-4367:
+        cml.assert_logged(msg=r"[^[]*Running: \[('git', 'annex'|'git-annex'), "
```

Comment:
Hmm, so this is a little removed from testing the actual behavioral issue in gh-4367, but, given this rearrangement fixes the issue, it should be good enough to prevent a regression.